dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Messages.properties

```diff
@@ -48,7 +48,14 @@ WIZARD_PAGE_SELECT_GENERATOR_TYPE=Select a generator type.
 BROWSE_DIALOG_MESSAGE_SELECT_DEFINITION=Select an OpenAPI definition from the project.
 BROWSE_DIALOG_TITLE_SELECT_DEFINITION=Select an OpenAPI definition file.
 
-BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder for the generated files.
-BROWSE_DIALOG_TITLE_SELECT_FOLDER=Select a folder.
+BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder under the current project
+BROWSE_DIALOG_OUTPUT_FOLDER_INVALID=Output Folder Invalid
+BROWSE_DIALOG_OUTPUT_FOLDER_INVALID_MESSAGE=The selected folder must be the project root folder or one of project's child folders.
 
-BROWSE_DIALOG_NO_CHILD_FOLDERS=This project does not contain child folders. The stub will be generated into the project root.
+MAVEN_PROJECT_DETECTED_TITLE=Overwrite Existing Files
+MAVEN_PROJECT_DETECTED_DESCRIPTION=Warning: Existing files might be overwritten and the existing pom.xml will be merged with the generated pom.xml from the code generator. The original pom.xml will be backed up. Do you want to continue?
+MAVEN_PROJECT_DIFFERENT_OUTPUT_FOLDER_DESCRIPTION=A child folder is selected. To build the generated code, manual refactoring is required to convert the Maven project into a multi-module project.
```

Comment:
Suggestion to remove passive voice in the second sentence:
To build the generated code, use manual refactoring to convert the Maven project into a multi-module project.

Reply:
Will make this change.

Reply:
Thanks!